From: David Bauer <mail@david-bauer.net>
Date: Sun, 21 Jan 2024 02:03:56 +0100
Subject: tools flex: disable parallel build

Sometimes the build of flex fails with erros like these:

./build-aux/config.sub: line 63: opying: command not found

Disable the parallel build of flex in order to avoid these issues from
occuring.

Signed-off-by: David Bauer <mail@david-bauer.net>

diff --git a/tools/flex/Makefile b/tools/flex/Makefile
index 177136b78c77c31c56db2fc993d1273d3dd81562..cb4f95e8d0c45a9c328c7ce48235a0ba80c02c6e 100644
--- a/tools/flex/Makefile
+++ b/tools/flex/Makefile
@@ -15,7 +15,6 @@ PKG_SOURCE_URL:=https://github.com/westes/flex/releases/download/v$(PKG_VERSION)
 PKG_HASH:=e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995
 
 HOST_FIXUP:=autoreconf
-HOST_BUILD_PARALLEL:=1
 
 include $(INCLUDE_DIR)/host-build.mk
 
